Library/Formula/magit.rb

```diff
@@ -37,22 +16,30 @@ class Magit < Formula
     sha256 "c528977bcfef51e5a93e52b68ece520973be1aaa0d8a44f5adda7c99bcfc87b0" => :mountain_lion
   end
 
+  depends_on :emacs => "24.4"
+
+  resource "dash" do
+    url "http://melpa.org/packages/dash-20150701.20.el"
+    sha256 "4d4e41c343fe0b3056d5572b7f15ba0212e9ecefd1d735eb6b403321d3b982f5"
+  end
+
   def install
-    if build.stable?
-      buildpath.install resource("git-commit-mode"),
-                        resource("git-rebase-mode")
-      system "make", "install", "PREFIX=#{prefix}"
-      (share/"emacs/site-lisp").install "git-commit-mode.el",
-                                        "git-rebase-mode.el"
-    else
-      # Can't run `make install` alone without ENV.j1:
-      # https://github.com/magit/magit/issues/1670
-      system "make"
-      system "make", "install", "PREFIX=#{prefix}"
-      resource("dash").stage do
-        (share/"emacs/site-lisp").install "dash-20150611.922.el" => "dash.el"
-      end
+    resource("dash").stage do
+      (share/"emacs/site-lisp").install "dash-20150701.20.el" => "dash.el"
     end
+    (buildpath/"config.mk").write <<-EOS
+      LOAD_PATH = -L #{buildpath}/lisp -L #{share}/emacs/site-lisp
+    EOS
+
+    args = %W[
+      PREFIX=#{prefix}
+      lispdir=#{share}/emacs/site-lisp
```

Comment:
I changed this from `#{share}/emacs/site-lisp/magit` to match other formulae, but is that correct?